app/src/main/java/com/kylecorry/trail_sense/tools/metaldetector/ui/FragmentToolMetalDetector.kt

```diff
@@ -173,6 +207,23 @@ class FragmentToolMetalDetector : BoundFragment<FragmentToolMetalDetectorBinding
             isVibrating = false
             haptics.off()
         }
+
+        updateMetalSoundIntensity(magneticField,metalDetected)
+    }
+    private fun updateMetalSoundIntensity(reading: Float, isMetalDetected: Boolean) {
+
+        if(isMetalDetected){
+            whistle.setVolume(0.0f)
+        }else{
+
+        }
```

Comment:
I believe the next few lines should be inside the else block

Reply:
oooo NOOO didn't commit 😊

Reply:
@kylecorry31  this logic don't need check for if metal is detected as        `  volume = SolMath.map(delta - threshold, 0f, 30f, 0f, 1f, true`) will  always return 0.0 if it is not detected

<img width="225" alt="Screenshot 2024-03-16 at 14 36 19" src="https://github.com/kylecorry31/Trail-Sense/assets/26343440/ace1e4fd-b59f-4024-ba59-8cffa0aaba6e">

Reply:
You are correct, that looks good - thanks!